docs/alerts/alert-rules.md

```diff
@@ -0,0 +1,163 @@
+
+### Rule Preview
+
+The **Rule Preview** chart visualizes the performance of the metric(s) that the alert rule monitors. This chart provides a live or recent historical view of the metric and shows when the conditions defined by the alert rule have been met.
+
+![Alert Rule Preview Chart](../images/alerts/alert-rule-preview-chart.png)
+
+It displays the relevant metric over time along with the threshold set by the alert rule. Hovering over specific points on the chart reveals detailed information.
+
+### Triggered Alert Events
+
+This section lists all triggered alert events that have occurred based on this alert rule.
+
+![Alert Rule Events](../images/alerts/alert-rule-events.png)
+
+The table includes:
+
+- **Alert Time:** The exact time when the alert was triggered. Clicking on this leads to the specific alert event details.
+- **Tags used for Grouping:** A list of tags by which the alert metric is grouped, such as `os.host`.
+
+This section helps you track historical alert events.
+
+### Rule Details
+
+The **Rule Details** section provides an quick view of the alert rule’s configuration.
+
+![Alert Rule Details](../images/alerts/alert-rule-details.png)
+
+Fields included:
+
+- **Metrics:** The specific metric(s) being monitored.
+- **Priority:** The priority of the alert rule
+- **Groups:** How data is grouped (e.g., by host).
+- **Interval:** The time interval before the alert triggers.
+- **Threshold:** The conditions that trigger the alert.
+- **Creator:** The creator of the alert rule.
+
+You can configure the alert rule by clicking the three vertical dots in the top right corner and selecting **Edit Alert Rule**.
+
+![Edit Alert Rule](../images/alerts/alert-rule-edit.png)
+
+## Creating an Alert Rule
+
+There are multiple methods to create an alert rule:
+
+1. **From a Chart:** Hover over any chart within an App report and click the small alert icon that appears at the top. This action opens the **'Create Alert Rule'** dialog, automatically populating properties such as filtering and grouping based on the settings applied within that specific chart.
+
+2. **From the App Report:** Navigate to any App report and click on the alert icon in the actions menu at the top of the report.
+
+3. **From the Alerts Page:** Access the Alerts page through the left menu panel and select **'+ New Alert Rule'** from the dropdown menu located at the top right of the screen.
+
+![Create Alert Rules](../images/alerts/create-alert-rules.gif)
+
+Once the **New Alert Rule** window opens, you can edit the rule name and select one of the available options (tabs) on the left to configure the alert, as detailed in the following subsections.
+
+### Alert Metric
+
+If you open the **New Alert Rule** window from a specific App, that App will be automatically selected in the dialog. Next to the App selection, a dropdown allows you to choose one or multiple metrics to receive alerts for.
```

Comment:
Wondering how we can make it clear this is all.... maybe the whole page really... all about Monitoring/Logs, and not e.g. Synth.

Reply:
We partially address this in the last section, **Additional Resources on Alert Rules**, where we provide a specific link for creating Synthetics alerts.